.github/workflows/release-elixir.yaml

```diff
@@ -18,7 +18,7 @@ jobs:
       - name: Download libextism
         uses: actions/download-artifact@v3
         with:
-          name: libextism-${{ matrix.os }}
+          name: libextism-ubuntu-latest
```

Comment:
`${{ matrix.os }}` is only valid if there is a strategy matrix defined in `job`: 
https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs

Reply:
I think it would still evaluate to an empty string though?

Reply:
I think i deleted this